### PR TITLE
Update landscape.yml to remove TF and add Nephio

### PR DESCRIPTION
**Pull request by wrathwolf**
Removed archived TF project and added new Nephio project.

### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [x] Is your project closed source or, if it is open source, does your project have at least 250 GitHub stars?
* [x] Have you picked the single best (existing) category for your project?
* [x] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [x] Have you included a URL for your SVG or added it to `hosted_logos` and referenced it there?
* [x] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] Does your project/product name match the text on the logo?
* [x] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [x] ~10 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?

### CODE REVIEW

File: landscape.yml [hunk 1]
```diff
@@ -486,67 +486,42 @@ landscape:
...
+              - repo_url: https://github.com/nephio-project/docs
+              - repo_url: https://github.com/nephio-project/catalog
+              - repo_url: https://github.com/nephio-project/nephio-packages
+              - repo_url: https://github.com/nephio-project/one-summit-22-prow-project
+              - repo_url: https://github.com/nephio-project/free5gc-packages
+              - repo_url: https://github.com/nephio-project/kpt-backstage-plugins
+              - repo_url: https://github.com/nephio-project/sig-release
+              - repo_url: https://github.com/nephio-project/edge-status-aggregator
+              - repo_url: https://github.com/nephio-project/sig-release
+              - repo_url: https://github.com/nephio-project/edge-status-aggregator
```
Comment (bramwelt) [inline landscape.yml:513]: These repositories are listed twice.

File: landscape.yml [hunk 2]
```diff
@@ -486,67 +486,42 @@ landscape:
...
+              - repo_url: https://github.com/nephio-project/api
+              - repo_url: https://github.com/nephio-project/nephio-example-packages
+              - repo_url: https://github.com/nephio-project/oai
+              - repo_url: https://github.com/nephio-project/free5gc
+              - repo_url: https://github.com/nephio-project/governance
+              - repo_url: https://github.com/nephio-project/nephio-sdk
+              - repo_url: https://github.com/nephio-project/docs
+              - repo_url: https://github.com/nephio-project/catalog
+              - repo_url: https://github.com/nephio-project/nephio-packages
+              - repo_url: https://github.com/nephio-project/one-summit-22-prow-project
```
Comment (bramwelt) [inline landscape.yml:507]: I believe two repos got merged under one name here some how. This should be
```
- repo_url: https://github.com/nephio-project/one-summit-22-workshop
- repo_url: https://github.com/nephio-project/nephio-test-prow-project
```

File: landscape.yml [hunk 3]
```diff
@@ -486,67 +486,42 @@ landscape:
...
-            homepage_url: https://tungsten.io/
-            logo: tungstenfabric-stacked-color.svg
+            name: Nephio
+            homepage_url: https://nephio.org
+            logo: Nephio_logo_colorblack_horizontal.svg
+            crunchbase: https://www.crunchbase.com/organization/nephio
             twitter: https://twitter.com/LF_Networking
-            crunchbase: https://www.crunchbase.com/organization/tungsten-fabric
-            repo_url: https://github.com/tungstenfabric/ci-management
+            repo_url: https://github.com/nephio-project
```
Comment (bramwelt) [inline landscape.yml:494]: This should be the url of a repository, not the project. I'd recommend changing this to `https://github.com/nephio-project/nephio` and removing `/nephio-project/nephio` from the `additional_repos` list.

**Review by bramwelt — CHANGES_REQUESTED** [review]

File: landscape.yml (hunk 3 above)
Comment (wrathwolf) [reply landscape.yml]: Thanks for catching the duplicates and the explanation of the URL.  I've resolved the things you've identified.

**Pull request merged**